src/Identity/Core/src/IdentityBuilderExtensions.cs

```diff
@@ -75,4 +78,19 @@ public static IdentityBuilder AddSignInManager(this IdentityBuilder builder)
         builder.Services.AddScoped(managerType, typeof(TSignInManager));
         return builder;
     }
+
+    private sealed class PostConfigureSecurityStampValidatorOptions : IPostConfigureOptions<SecurityStampValidatorOptions>
```

Comment:
Maybe add a comment on this class (and others) that it exists to set TimerProvider property because the type doesn't have a constructor?